### PR TITLE
Build Json fetching 

### PR DESCRIPTION
**Pull request by chen678**
The server will return hard-coded ArrayList as fake comments data, and `/js/script.js` should be fetching this data and using it to add HTML elements to the page. The console will print this date as well to show its reception.

Web page screenshot:
Screen Shot 2020-02-27 at 7 04 57 PM
Screen Shot 2020-02-27 at 7 05 04 PM

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,19 +14,40 @@
...
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    String json = convertToJson(createArrayList());
+
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println(json);
+  }
+
+  private ArrayList<String> createArrayList(){
```
Comment (kmontgomery) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Can you add a more descriptive name to this method?

File: portfolio/src/main/webapp/js/script.js [hunk 2]
```diff
@@ -12,17 +12,22 @@
...
+function responsiveTopNav() {
+    var x = document.getElementById("myTopnav");
+    if (x.className === "topnav") {
+        x.className += " responsive";
+    } else {
+        x.className = "topnav";
+    }
 }
+
+function myFetch(){
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/js/script.js]: Add javadoc describing what this method does to help the reader understand.

File: portfolio/src/main/webapp/js/script.js [hunk 3]
```diff
@@ -12,17 +12,22 @@
...
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Todo: implement a working responsive top navigation tabs
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/js/script.js:17]: Nice job leaving a TODO!

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,86 @@
...
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  
+  <body onload = "myFetch()">
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/index.html]: Rename this method so that the reader can have a better understanding of what it does.

**Review by kmontgomery — CHANGES_REQUESTED** [review]
Great work!

**Comment by chen678** [comment]
addressed comments. 
Screen Shot 2020-02-27 at 9 06 05 PM

**Review by kmontgomery — APPROVED** [review]

**Comment by chen678** [comment]
@kmontgomery thanks for reviewing @night!

**Pull request merged**